### DISCUSSION TITLE
Google Api Error: Invalid request - Precondition check failed.

### DISCUSSION THREAD

**Discussion by NareshSethu** [fastlane/fastlane#22184] [Q&A]
We are using Fastlane to spin Android apps while doing so and uploading the app using supply lane to one of the tracks in Google Play Console we are getting the below error 

**Google Api Error: Invalid request - Precondition check failed.**

<Google::Apis::AndroidpublisherV3::Track:0x0000000003ba5970 @track="beta">, #<Google::Apis::AndroidpublisherV3::Track:0x0000000003b9ce60 @track="alpha">, #<Google::Apis::AndroidpublisherV3::Track:0x0000000003b97578 @track="internal", @releases=[#<Google::Apis::AndroidpublisherV3::TrackRelease:0x0000000003b959f8 @name="2 (24.2.0)", @status="draft", @version_codes=["2"]>]>]

Could any of you help me on this regard.

**Comment by marioprieta** [comment]
That happens because the track name is not correct
